docs/src/assets/scss/_docs-footer.scss

```diff
@@ -7,6 +7,7 @@
 
     @media all and (max-width: 800px) {
```

Comment:
Try setting your view width to exactly 800px. It'll give a combination of the tablet and desktop views:

![Screen Shot 2022-06-01 at 1 05 08 AM](https://user-images.githubusercontent.com/1709537/171331814-6579724c-b947-4cc0-a906-51b54a83c426.png)

That's because there's a `min-width: 800px` media query that makes `.docs-content` a grid.

Currently, the footer is centered at widths <= 800px, and the page is a grid at >= 800px, so there's a bit of overlap. If we make this a max-width of 799px, that overlap goes away:

```suggestion
    @media all and (max-width: 799px) {
```